tests/stubs/openai.ts

```
@@ -0,0 +1,27 @@
+                        export interface ChatCompletionToolChoiceOption {}
+                }
+        }
+        export namespace ChatCompletionCreateParams {
+                export interface Function {
+                        name: string;
+                        description?: string;
+                        parameters?: unknown;
+                }
+        }
+        export namespace Responses {
+                export interface Tool {}
```

Biome (2.1.2)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)